app/page.tsx

```diff
@@ -50,15 +51,15 @@ export default function Home() {
             Craft content effortlessly with integrated AI features
           </h1>
           <Image
-            src={"/assets/ai-clip.png"}
+            src="/assets/ai-clip.png"
```

Comment:
I see you have created a dark mode version of this image but haven't added it here. Can we have that image when it is in dark mode?

Reply:
We can add that, but then we also need to add "use client" to page.tsx due to the use of useState hook for the dark mode. But it is not allowed due to the metadata (title and description) present in the same file. We need to move the metadata directly to layout.tsx or maybe we can also create a separate file. 
So I just kept the dark mode image and didnt add it into the code without your permission.
I was asking for discord for the same.

Reply:
> We can add that, but then we also need to add "use client" to page.tsx due to the use of useState hook for the dark mode. But it is not allowed due to the metadata (title and description) present in the same file. We need to move the metadata directly to layout.tsx or maybe we can also create a separate file. So I just kept the dark mode image and didnt add it into the code without your permission. I was asking for discord for the same.

Also, I couldn't recreate the dark mode version of hero.png file, I thought it will be better to have all the imaged either dark mode or light mode to maintain the consistency.